test/selenium/sessions_ui_test.js

```diff
@@ -7,13 +7,17 @@ var webdriver = require('selenium-webdriver'),
     assert    = require('assert'),
     util      = require('util');
 
+var Prepper = require('./lib/testPrepperSelenium.js');
+
 var HOST_URL = util.format(
     'http://%s:%s/',
     process.env.BOX_LOCAL_NAME && process.env.BOX_LOCAL_NAME != 'localhost' ?
         (process.env.BOX_LOCAL_NAME + '.local.') : 'localhost',
     process.env.BOX_PORT || '3000'
 );
 
+Prepper.makeSuite('Test set up UI',function(){
+
 describe('sessions ui', function() {
   var driver;
   this.timeout(8000);
```

Comment:
I don't think you'll need this timeout here, because the Prepper has the long timeout value before the describe() call